src/components/about/About.js

```diff
@@ -114,31 +113,30 @@ const About = () => {
               </span>
             </h2>
             <p>
-              The goal of the Twitterbot on the Blue Witness project is to
-              adequately scrape twitter for reports of incidents of police
+              The goal of the Twitter bot on the Blue Witness project is to
+              adequately scrape Twitter for reports of incidents of police
               violence. This is a complicated process, and inevitably some data
-              is gathered that is at the very least difficult to verify or at
-              the very worst simply not related to police incidents at all. As
-              the Blue Witness project grows, the team is refining the training
-              of this model to not only more accurately reflect individual
-              incidents on twitter but to reach out to people posting those
-              incidents and automatically ask some follow-up questions,
-              enhancing the Human Rights First administrative teams ability to
-              quickly update the incident map as well as the data available for
-              download to researchers, journalists, students and activists. If
-              you'd like to know more, feel free to reach out to HRF on our
-              contact page.
+              gathered will be difficult to verify or at the very worst simply
+              not relate to police incidents at all. As the Blue Witness project
+              grows, the team is refining the training of this model to
+              accurately reflect individual incidents on Twitter. It reaches out
+              to people posting those incidents and automatically ask some
+              follow-up questions, enhancing the Human Rights First
+              administrative team. By having the Ability to quickly update the
+              incident map as well as the data available for download to
+              researchers, journalists, students, and activists. If you'd like
+              to know more, feel free to reach out to HRF on our contact page.
             </p>
           </div>
         </div>
         <div className="inside-bottom-container lambda-credits-container">
           <div className="lambda-credits">
             <h2>Who Built This Website</h2>
             <p>
-              This project was designed and built in partnership between Human
+              This project designed and built in partnership between Human
               Rights First's Innovation Lab and Lambda School. It represents
-              nearly seven months of work by Lambda School students. Below are
-              all the students, and their Github profiles, that have ever worked
+              nearly eight months of work by Lambda School students. Below are
+              all the students, and their GitHub profiles, that have ever worked
```

Comment:
Suggesting:
"Below are the students who have worked on this project with links to their Github profiles included.